packages/@aws-cdk/aws-stepfunctions-tasks/lib/emr/emr-create-cluster.ts

```diff
@@ -34,119 +40,109 @@ export interface EmrCreateClusterProps {
 
   /**
-   * An IAM role for automatic scaling policies. A Role will be created if one is not provided.
+   * An IAM role for automatic scaling policies.
    *
-   * @default No autoScalingRole
+   * @default - A role will be created.
    */
   readonly autoScalingRole?: iam.IRole;
 
   /**
    * A list of bootstrap actions to run before Hadoop starts on the cluster nodes.
    *
-   * @default No bootstrapActions
+   * @default - None
    */
   readonly bootstrapActions?: EmrCreateCluster.BootstrapActionConfigProperty[];
 
   /**
    * The list of configurations supplied for the EMR cluster you are creating.
    *
-   * @default No configurations
+   * @default - None
    */
   readonly configurations?: EmrCreateCluster.ConfigurationProperty[];
 
   /**
    * The ID of a custom Amazon EBS-backed Linux AMI.
    *
-   * @default No customAmiId
+   * @default - None
    */
   readonly customAmiId?: string;
 
   /**
-   * The size, in GiB, of the EBS root device volume of the Linux AMI that is used for each EC2 instance.
+   * The size of the EBS root device volume of the Linux AMI that is used for each EC2 instance.
    *
-   * @default EMR selected default
+   * @default - EMR selected default
    */
-  readonly ebsRootVolumeSize?: number;
+  readonly ebsRootVolumeSize?: cdk.Size;
 
   /**
    * Attributes for Kerberos configuration when Kerberos authentication is enabled using a security configuration.
    *
-   * @default No kerberosAttributes
+   * @default - None
    */
   readonly kerberosAttributes?: EmrCreateCluster.KerberosAttributesProperty;
 
   /**
    * The location in Amazon S3 to write the log files of the job flow.
    *
-   * @default No logUri
+   * @default - None
    */
   readonly logUri?: string;
 
   /**
    * The Amazon EMR release label, which determines the version of open-source application packages installed on the cluster.
    *
-   * @default EMR selected default
+   * @default - EMR selected default
    */
   readonly releaseLabel?: string;
 
   /**
    * Specifies the way that individual Amazon EC2 instances terminate when an automatic scale-in activity occurs or an instance group is resized.
    *
-   * @default EMR selected default
+   * @default - EMR selected default
    */
   readonly scaleDownBehavior?: EmrCreateCluster.EmrClusterScaleDownBehavior;
 
   /**
    * The name of a security configuration to apply to the cluster.
    *
-   * @default No securityConfiguration
+   * @default - None
    */
   readonly securityConfiguration?: string;
 
   /**
    * A list of tags to associate with a cluster and propagate to Amazon EC2 instances.
    *
-   * @default No Tags
+   * @default - None
    */
-  readonly tags?: cdk.CfnTag[];
+  readonly tags?: { [key: string]: string };
 
   /**
    * A value of true indicates that all IAM users in the AWS account can perform cluster actions if they have the proper IAM policy permissions.
    *
-   * @default true
+   * @default - true
```

Comment:
The hyphen is added only when the default value needs to be described, rather than specified.

In this case, it should be 

```suggestion
@default true
```

Reply:
good catch, updated